hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/datanode/metrics/DataNodeMetrics.java

```diff
@@ -188,6 +188,15 @@
   @Metric MutableCounterLong packetsSlowWriteToDisk;
   @Metric MutableCounterLong packetsSlowWriteToOsCache;
 
+  @Metric("Number of replaceBlock ops between" +
+      " storage types on same host with local copy")
+  private MutableCounterLong replaceBlockOpOnSameHostWithCopy;
+  @Metric("Number of replaceBlock ops between" +
+      " storage types on same disk mount using hardlink")
+  private MutableCounterLong replaceBlockOpOnSameHostWithHardlink;
```

Comment:
Both "withHardlink" and "withCopy" are our block movement implementation. If in the future we change our implementation these names may no long hold. How about we change the metric names to "OnSameHost" and "OnSameMount" ? But then we need to think more about their semantic meanings. Maybe "OnSameHost" also includes the "OnSameMount"... Thoughts?

Reply:
Yeah sounds good, OnSameHost can include OnSameMount actually. I will make the change.